File: 551-StudentAttendanceRecordI/551-StudentAttendanceRecordI.py
# Last updated: 10/08/2026, 02:34:47
class Solution:
    def checkRecord(self, s: str) -> bool:
        count = 0
        flag = False

        for i in range(len(s)):
            if s[i] == 'A':
                count += 1
            if i < len(s) - 2 and s[i] == 'L' and s[i+1] == 'L' and s[i+2] == 'L':
                flag = True

        return count < 2 and not flag

File: 551-StudentAttendanceRecordI/551-StudentAttendanceRecordI.cpp
// Last updated: 10/08/2026, 02:34:52
class Solution {
public:
    bool checkRecord(string s) {
        int count = 0;
        int flag = 0;
        for(int i = 0; i<s.size(); i++){
            if(s[i] == 'A') count++;
            if(s[i] == 'L' && s[i+1] == 'L' && s[i+2] == 'L') flag = 1;
        }
        if(count<2 && flag != 1) return true;
        else return false;
    }
};